EnglishFilter/sections/antiadblock.txt

```diff
@@ -97,6 +97,7 @@ fckdescargas.com.ve,indoblog.me,balog18.blogspot.com,yupptv.in,fforfree.net,300m
 @@||thefilmbay.com^$generichide
 @@||pipocas.tv^$generichide
 @@||tamilyogi.nu^$generichide
+@@||satoshilabs.net^$generichide
```

Comment:
Please add restriction to "non Android/Windows/Mac app and Chrome extension" and add domain to usual BlockAdBlock javascript rule.

Reply:
@BlazDT it is not required
`!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)`